### ISSUE TITLE
To preserve accuracy in training set for datasets without test set

### ISSUE THREAD

**Issue by micheng-ZJU** [alibaba/pipcook#585]
For datasets without test set, e.g., mock data, we would like to have an evaluation to tell the accuracy from training set. 
Both image classification models and object detection models.

**Comment by yorkie** [comment]
@FeelyChau @rickycao-qy This requires we define the evaluation in more detail and save them in the database.
